Makefile

```diff
@@ -304,10 +304,11 @@ DATAPLANE_REGISTRY_URL                           ?=quay.io/podified-antelope-cen
 DATAPLANE_CONTAINER_TAG                          ?=current-podified
 DATAPLANE_OVN_METADATA_AGENT_BIND_HOST           ?=127.0.0.1
 DATAPLANE_SINGLE_NODE                            ?=true
-DATAPLANE_KUTTL_CONF      ?= ${OPERATOR_BASE_DIR}/dataplane-operator/kuttl-test.yaml
-DATAPLANE_KUTTL_DIR       ?= ${OPERATOR_BASE_DIR}/dataplane-operator/tests/kuttl/tests
-DATAPLANE_KUTTL_NAMESPACE ?= dataplane-kuttl-tests
-DATAPLANE_DEFAULT_GW      ?= 192.168.122.1
+DATAPLANE_KUTTL_CONF                             ?= ${OPERATOR_BASE_DIR}/dataplane-operator/kuttl-test.yaml
+DATAPLANE_KUTTL_DIR                              ?= ${OPERATOR_BASE_DIR}/dataplane-operator/tests/kuttl/tests
+DATAPLANE_KUTTL_NAMESPACE                        ?= dataplane-kuttl-tests
+DATAPLANE_DEFAULT_GW                             ?= 192.168.122.1
+DATAPLANE_CTLPLANE_INTERFACE                     ?= enp1s0
```

Comment:
Would BM_CTLPLANE_INTERFACE would make sense(?) Both dataplane and ctlplane maybe confusing?

Reply:
Fair enough.  Done.